src/components/DisplayContribution.tsx

```diff
@@ -166,7 +166,7 @@ const DisplayContribution = ({ tasks }) => {
           </View>
         ))
       ) : (
-        <Text style={{color: '#000'}}>No Tasks Yet!</Text>
+        <Text style={{ color: '#000' }}>No Tasks Yet!</Text>
```

Comment:
+1

Reply:
Happend because of prettier, Ill disable.